tests/lib/nested.sh

```diff
@@ -6,16 +6,26 @@
 # shellcheck source=tests/lib/systems.sh
 . "$TESTSLIB"/systems.sh
 
+# shellcheck source=tests/lib/store.sh
+. "$TESTSLIB"/store.sh
+
 NESTED_WORK_DIR="${NESTED_WORK_DIR:-/tmp/work-dir}"
 NESTED_IMAGES_DIR="$NESTED_WORK_DIR/images"
 NESTED_RUNTIME_DIR="$NESTED_WORK_DIR/runtime"
 NESTED_ASSETS_DIR="$NESTED_WORK_DIR/assets"
 NESTED_LOGS_DIR="$NESTED_WORK_DIR/logs"
 
+
 NESTED_VM=nested-vm
 NESTED_SSH_PORT=8022
 NESTED_MON_PORT=8888
 
+NESTED_CUSTOM_MODEL="${NESTED_CUSTOM_MODEL:-}"
```

Comment:
You need to add these vars in the tests suites env in the spread.yaml, otherwise the vars are not imported.

Reply:
Those can be added at global environment in the spread.yaml, not needed to add that to any nested suite

Reply:
as discussed on IRC, it's fine to leave this as-is without adding to spread.yaml since we aren't yet using these vars anywhere else except one specific test that is still proposed and not merged yet, we can move this definition around to spread.yaml's environment too when we have a need for this